modules/flags/FeatureFlags.js

```diff
@@ -28,6 +28,7 @@ class FeatureFlags {
 
         logger.info(`Source name signaling: ${this._sourceNameSignaling},`
             + ` Send multiple video streams: ${this._sendMultipleVideoStreams},`
+            + ` SSRC rewriting supported: ${this._ssrcRewriting},`
```

Comment:
Same comment about the ssrc rewriting support check, `sSsrcRewritingSupported() `needs to check if source-name signaling hasn't been disabled explicitly through config.js.

Reply:
fixed.